pkg/config/schema.go

```diff
@@ -36,12 +36,44 @@ type Logs struct {
 	Colors  bool `yaml:"colors" json:"colors" mapstructure:"colors"`
 }
 
+type Command struct {
+	Name        string            `yaml:"name" json:"name" mapstructure:"name"`
+	Description string            `yaml:"description" json:"description" mapstructure:"description"`
+	Env         []CommandEnv      `yaml:"env" json:"env" mapstructure:"env"`
+	Arguments   []CommandArgument `yaml:"arguments" json:"arguments" mapstructure:"arguments"`
+	Flags       []CommandFlag     `yaml:"flags" json:"flags" mapstructure:"flags"`
+	Steps       []string          `yaml:"steps" json:"steps" mapstructure:"steps"`
+	Commands    []Command         `yaml:"commands" json:"commands" mapstructure:"commands"`
+}
+
+type CommandArgument struct {
+	Name        string `yaml:"name" json:"name" mapstructure:"name"`
+	Description string `yaml:"description" json:"description" mapstructure:"description"`
+}
+
+type CommandFlag struct {
+	Name        string `yaml:"name" json:"name" mapstructure:"name"`
+	Shorthand   string `yaml:"shorthand" json:"shorthand" mapstructure:"shorthand"`
+	Type        string `yaml:"type" json:"type" mapstructure:"type"`
+	Description string `yaml:"description" json:"description" mapstructure:"description"`
+	Usage       string `yaml:"usage" json:"usage" mapstructure:"usage"`
+	Required    bool   `yaml:"required" json:"required" mapstructure:"required"`
+}
+
+type CommandEnv struct {
+	Key          string `yaml:"key" json:"key" mapstructure:"key"`
+	Value        string `yaml:"value" json:"value" mapstructure:"value"`
+	ValueCommand string `yaml:"valueCommand" json:"valueCommand" mapstructure:"valueCommand"`
+}
+
 type Configuration struct {
-	BasePath   string `yaml:"base_path" json:"base_path" mapstructure:"base_path"`
-	Components Components
-	Stacks     Stacks
-	Workflows  Workflows
-	Logs       Logs
+	BasePath    string `yaml:"base_path" json:"base_path" mapstructure:"base_path"`
+	Components  Components
+	Stacks      Stacks
+	Workflows   Workflows
```

Comment:
This is consistent where both the left and right columns contain the same plural form (inconsistent with the above).

Reply:
`Workflows` type is more complicated, don't want to change workflows in this unrelated PR.

Reply:
Workflows is a diff type from `Commands` and `Arguments`

It's a map
```
BasePath string `yaml:"base_path" json:"base_path" mapstructure:"base_path"`
```